docs/presentations/articles/virtual_assistants.md

```diff
@@ -0,0 +1,104 @@
+# Virtual Assistants
+
+
+_Author: [Marco Mollo](mailto:ma161mol@htwg-konstanz.de)_
+
+## TL;DR
+
+Virtual assistants like Siri, Alexa, and Google Assistant rely on Natural Language Processing (NLP) for seamless interactions.
+
+## Introduction
+
+This articel will feature the underlying technology, how virtual assistants work, and how we as developers can use them.
+
+## Underlying Technologies
+
+1. Speech Recognition
+2. NLP
+3. Dialog Management
+4. Machine Learning & AI
+5. Cloud-Based Processing
+6. Wake Word Detection
+7. Integration with Services
+8. Privacy and Data Security
```

Comment:
You can omit this list I'd say, it's generated in the outline to the right anyways